### PR TITLE
Credentials Table and Footer Navigation

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#13386

*Description of changes:*
* Implements default Credentials table with mocked Data* To be replaced with Credentials GET once available.
* Implements a Credentials Footer to support Step and completion navigation. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by vysakh-menon-aot — APPROVED** [review]

File: src/assets/styles/theme.scss [hunk 1]
```diff
@@ -27,6 +27,7 @@ $BCgovABlue1: #6e93ff;
 $BCgovABlue2: #3b6cff;
 $BCgovABlue3: #0846ff;
 $BCgovABlue4: #003bee;
+$BCgovBlue5O: #01336626; // LOW OPACITY
```
Comment (severinbeauvais) [inline src/assets/styles/theme.scss:30]: I thought hex colors had to be 6 (or 3) chars. Does this work?

**Review by severinbeauvais — APPROVED** [review]
LGTM

File: src/assets/styles/theme.scss (hunk 1 above)
Comment (cameron-freshworks) [reply src/assets/styles/theme.scss:30]: Indeed it does!
On the web, you can add the transparency code to the end of the hex, so in this case `26` will apply a 15% transparency, i think for mobile the code goes at the start?
Forgive me, i can't recall the exact origination of the colour but i imagine it was pulled directly from some previous design specs for assets and copied over to the uxpin for the credentials pilot.

Comment (cameron-freshworks) [reply src/assets/styles/theme.scss:30]: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Comment (severinbeauvais) [reply src/assets/styles/theme.scss:30]: That gist shows the transparency code at the front...?

Comment (severinbeauvais) [reply src/assets/styles/theme.scss:30]: Suggestion: update comment to indicate level of opacity or format (for when I look at this again and have forgotten everything).

I leave it in your capable hands :)

Comment (cameron-freshworks) [reply src/assets/styles/theme.scss:30]: haha totally, will do!

**Pull request merged**